src/app/shared/entity/provider/aws.ts

```diff
@@ -73,4 +73,5 @@ export class AWSSize {
 export enum Architecture {
   ARM64 = 'arm64',
   X64 = 'x64',
+  X86_64 = 'x86_64',
```

Comment:
I don't think we need to define architecture here this is for KKP managed clusters.
shouldn't be part of `src/app/shared/entity/provider/eks.ts` file?